PowerPointLabs/PowerPointLabs/HighlightTextFragments.cs

```diff
@@ -41,7 +41,12 @@ public static void AddHighlightedTextFragments()
                 if (selectedText.Length <= 0) return;
                 if (selectedShapes.Count != 1) return;
 
-                List<PowerPoint.Shape> selectionToAnimate = GetShapesFromLinesInText(currentSlide, selectedText, selectedShapes[1]);
+                 if (selectedShapes.Fill.ForeColor.RGB == 16777215)

```

Comment:
what is 16777215?

can move all these codes into function GetShapesFromLinesInText?

and about the naming, for a boolean var it should'd better start with is. like
isHasBackground = selectedShapes.Fill.ForeColor.RGB == ...